webapp/components/definition/populations/NetPyNEPopulation.js

```diff
@@ -160,23 +160,23 @@ class NetPyNEPopulation extends React.Component {
           newValue,
           (response, newValue) => {
             this.renaming = false;
-            this.props.updateCards();
+            // this.props.updateCards();
```

Comment:
if this has to return in the feature, better leave a comment on why it was commented. If it's not needed anymore, better just remove it as the git history will give us all we need